testcontainers-dapr/src/main/java/io/dapr/testcontainers/DaprContainer.java

```diff
@@ -303,11 +353,33 @@ protected void configure() {
       this.schedulerContainer.start();
     }
 
+    dependsOn(placementContainer, schedulerContainer);
+  }
+
+  @Override
+  public void start() {
     List<String> cmds = new ArrayList<>();
     cmds.add("./daprd");
+    
+    String finalAppName = (appName != null && !appName.trim().isEmpty())
```

Comment:
@cicoyle I like this, but I think before the `appName` was mandatory, as if it was null it should break the container to start.